packages/back-end/src/models/ReportModel.ts

```diff
@@ -118,6 +118,7 @@ export async function updateReport(
       $set: {
         ...updates,
         dateUpdated: new Date(),
+        error: updates.error || "",
```

Comment:
I don't think this is right place for this.  If someone just edits the report name, this will now delete the error.  I think this belongs in the ReportQueryRunner class instead.  That's where we call `updateReport`

Reply:
Ah, yea - good call. Updated.